server.js

```diff
@@ -28,9 +29,17 @@ if (process.env.NODE_ENV === 'development') {
 // Mount routes
 app.use(CAMPAIGNS_ROUTE, campaigns);
 
+app.use(errorHandler);
```

Comment:
Not important, but there is a comment `// Middleware` above for middleware section. It's better to move that there, and keep this section for routes only.

Reply:
Moved the comment down since error handler should be added after the rout (mentioned in video).